### PR TITLE
Extend default options rather than assign with provided options

### PR DESCRIPTION
**Pull request by szmudo**
[FIX] less: less default options should rather be extended

the less default options have been overruled by the provided consumer options which results in exceptions due to missing default options

Now the default options are extended with the provided consumer options

### CODE REVIEW

File: lib/less.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ var fs = require('fs');
 var url = require('url');
 var path = require('path');
 var async = require('async');
-var assign = require('object-assign');
```
Comment (matz3) [inline lib/less.js]: As `object-assign` is not used anymore, please also remove it from the dependencies.

Comment (szmudo) [inline lib/less.js]: removed `object-assign` dependency

**Comment by matz3** [comment]
Thanks! :+1:
I will release a patch today.

**Comment by szmudo** [comment]
Thanks a lot :+1:

**Comment by matz3** [comment]
`0.4.1` is now published

**Pull request merged**